utils/src/config.rs

```diff
@@ -0,0 +1,582 @@
+use getset::Getters;
+use serde::Deserialize;
+use std::{collections::HashMap, path::PathBuf};
```

Comment:
use std before use third-party crates for same code style

Reply:
Done